resp/get_foxsi_resp.pro

```diff
@@ -0,0 +1,84 @@
+FUNCTION get_foxsi_resp, det = det, bin = bin, erange = erange, fwhm = fwhm, $
+	offaxis_arcmin=offaxis_arcmin, save_file = save_file, type = type, $
+	foxsi1 = foxsi1, _extra = _extra 
+
+;
+; PURPOSE: Produce FOXSI instrument response for single detector-optic pairing which 
+;          incorporates finite energy resolution of the detectors.
+;
+; KEYWORDS: 
+;	DET:		set detector number to use (0-6)
+;	BIN: 		energy bin size (keV)
+;	ERANGE: 	energy range, include start and end values (keV)
+;	FWHM:		energy resolution of detector (0.5 keV for Si, 1.0 keV for CdTe)
+;	OFFAXIS_ARCMIN:	offaxis position of source (arcmin) 
+;	SAVE_FILE: 	set this keyword to save the response in an IDL .sav file
+;	TYPE: 		set detector type ('si' or 'cdte')
+;	FOXSI1: 	set this keyword for FOXSI-1 response (default is FOXSI-2)
```

Comment:
The usual way in other routines is to have a keyword "year", which can be 2012, 2014 (and soon 2018), to distinguish between foxsi-1 and foxsi-2. The default is 2014. I think it would be great to continue doing so, and also this provides the opportunity to use this routine for more than just foxsi-1 and foxsi-2 in later iterations

Reply:
actually, also, I did not find where the foxsi1 keyword is used in the routine.

Reply:
I like the keyword 'year' better as well, but the keyword 'foxsi1' is currently used for get_foxsi_effarea.pro, which is utilized in this routine (default is to use FOXSI-2 values). So maybe we can include that as part of the upgrades for FOXSI-3?

And you're right! I added the foxsi1 keyword last-minute, and it didn't make it into the get_foxsi_effarea call. I have since added that!